### ISSUE TITLE
Alt text is missing or not meaningful in describing the image

### ISSUE THREAD

**Issue by saad-mo** [SAP/spartacus#11319]
Current Narration

Not Meaningful
![Screen Shot 2020-09-28 at 5 20 15 PM](https://user-images.githubusercontent.com/15215400/135929920-e8b70c8d-9128-47df-af72-3edfb6eafbb6.png)

Absent
![Screen Shot 2020-09-28 at 5 20 35 PM](https://user-images.githubusercontent.com/15215400/135929934-2dbab931-3f5a-4bc7-9e90-198937bfdbfc.png)

Expectation

- Alternative text should describe the image in no more than 125 characters.
- If a description is longer than 125 characters needed, then the image should have a "longdesc" attribute, otherwise, it will linger even if the user moves to the next item.
- A good rule of thumb to consider is to include what you might relay over the phone.
- The word "image" is vocalized by default at the end of the alt text so no need to include it.

**Comment by saad-mo** [comment]
Updated ticket description to include the information from wiki link

**Comment by saad-mo** [comment]
- alt text being absent is due to incomplete sample data. 
- alt text that is not meaningful is also due to how the sample data is set up for some products. 
- The only thing we could do with this ticket is to check if `product.summary` is longer than 125 characters and render `longdesc` attribute instead of `alt`
- truncate  `product.summary` to 125 characters when it is used in product image alt text 

@developpeurweb any thoughts?

**Comment by developpeurweb** [comment]
@saad-mo 

- Let's set up a fallback for absent alternative text. **There must be text to avoid failing the auto-test.** The product name seems like an appropriate fallback.
- Since this is a demo meant to be filled in by customers, let's leave meaningfulness to be determined by them for each case and industry.
- `longdesc` should point to an alternative different file, so it may not be an option in this case.
- The summary, in this case, seems more like a feature pitch, and doesn't really describe the image. I'd rather go with the **product name as a fallback** in case of missing alternative text. Even though 125 characters is the best practice, let's keep it unrestricted.

**Comment by KateChuen** [comment]
Tested on develop branch using yarn.

Now, the alt text of an image uses the product name. 
<img width="1304" alt="Screen Shot 2022-04-22 at 11 50 08 AM" src="https://user-images.githubusercontent.com/45409515/164749916-df60817b-1127-4380-9c5c-43a35b8e24ba.png">

<img width="1306" alt="Screen Shot 2022-04-22 at 11 45 49 AM" src="https://user-images.githubusercontent.com/45409515/164749705-07f926c0-bb0c-4823-971d-5ab7a81a38f2.png">

![ImagePLP](https://user-images.githubusercontent.com/45409515/164750047-a802fbc1-9f6d-4bd2-8150-a24ac6a27c7b.png)
